lib/src/arithmetic/floating_point/floating_point_utilities.dart

```diff
@@ -49,4 +49,22 @@ abstract class FloatingPointUtilities {
 
     return (larger, smaller);
   }
+
+  /// Sort two [FloatingPoint]s and swap them if necessary so that the one
+  /// the larger exponent is the first element in the returned tuple.
+  static (FpType larger, FpType smaller)
```

Comment:
wondering if we should ditch this utilities class and make these into their own components with port boundaries?

Reply:
Done.